finance/proposal-calls/internal_template.md

```diff
@@ -0,0 +1,56 @@
+# Phase II funding proposals
+
+*This is a draft document for how the Interim Finance Committee envisions the process of distributing grant money (e.g. from the Moore grant) within the Astropy project. This is a general template and the finance committee will change the details for individual calls. In particular, actual dollar numbers and details of the grant and grant rules that limit the work that could be proposed will need to be filled in. This template is written with the current Moore grant in mind.*
+
+## Proposal goals
+Proposals shall improve the Astropy Project. This includes code for the core repository, for coordinated or affiliated packages, infrastructure (e.g. testing, CI), tutorials, workshops, notebooks, community engagement, inclusion, diversity and empowerment, and anything else broadly Project related. 
+
+Proposals shall foster the Astropy Project goals of community, shared work, and cooperation and shall be proposed and executed in this manner.
+
+Total funding for this call is coming from (**link to grant here**) and XXX $ are available in total over XX months. 
+
+Proposals need to clearly identify what the funding is used for (e.g. "pay YYY $ to add Klingon Standard Time to astropy.time" or "organize astropy workshop to engage amateur astronomer community in Antarctica")
+
+## Proposal details
+**Fill in here when using this template to issue an internal Astropy call**
+
+- Source of money (e.g. Moore grant)
+- Specific area of proposal (e.g. Astropy may have a grant that can be used only for a specific purpose, e.g. only for community outreach)
+- Restrictions: Depending on who provided the money to astropy, it’s use might be restricted, e.g. only to contractors in certain countries
+
+## Proposal process
+Proposals should be posted as pull requests to github.com/astropy/astropy-project/ with the initial title text "Phase II Proposal:".
```

Comment:
Phase II Proposal -> isn't that specific to the Moore grant? If this is a template, should be a placeholder title?

Reply:
Yes, but we don't have that many calls that we'll use cookiecutter to fill in this template. I anticipate absolutely minor re-writing to happen every time anyway.